### PR TITLE
DOCO-752 Drupal 10 SimpleSAMLphp Compatibility

### LINKED ISSUE #7973: Using SimpleSAMLphp with Shibboleth SSO Doc Update
Re: [Using SimpleSAMLphp with Shibboleth SSO](https://pantheon.io/docs/shibboleth-sso)

Priority: Medium

## Issue Description:

With the D10.0.0 release just weeks away, it might be worth pointing out that the recommended solution for site-level SSO for Drupal on Pantheon is incompatible with D10/Symfony 6.

## Suggested Resolution

See https://github.com/simplesamlphp/simplesamlphp/issues/1689

### PR DESCRIPTION
**Pull request by whitneymeredith**
Closes #7973 

## Summary

**[Using SimpleSAMLphp with Shibboleth SSO](https://pantheon.io/docs/shibboleth-sso)** - Added Drupal 10 compatibility information.

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by whitneymeredith** [comment]
[Preview](https://pr-7987--pantheon-docs.my.pantheonfrontend.website/docs/shibboleth-sso#drupal-configuration)

**Comment by whitneymeredith** [comment]
@kporras07 Can you please review this?

File: source/content/shibboleth-sso.md [hunk 1]
```diff
@@ -195,18 +195,24 @@ You can now visit the subdirectory `/simplesaml` on your development site and co
...
 
 ```php:title=settings.php
 # Provide universal absolute path to the installation.
 $conf['simplesamlphp_auth_installdir'] = $_ENV['HOME'] .'/code/private/simplesamlphp';
 ```
 
-</Alert>
+### Drupal 10
+
+Drupal 10/Symfony 6 is incompatible with the recommended site-level SSO solution for Pantheon Drupal sites outlined in this document. However, the 4.x branch (Dev version) of the Drupal module can use the dev-master version of the SimpleSAMLphp library to achieve Drupal 10 compatibility.
```
Comment (kporras07) [inline source/content/shibboleth-sso.md:215]: The simplesamlphp library is not yet compatible with Symfony 6 (the Symfony version included in Drupal 10) except on the dev-master branch. There is a new simplesamlphp_auth branch (4.x) that you could use if you need this solution in Drupal 10.

To use it, require the Drupal module like this:

```
composer require drupal/simplesamlphp_auth:"^4"
```

Remember this means using dev versions so use it at your own risk.



@whitneymeredith what about something like this? ^

Comment (whitneymeredith) [reply source/content/shibboleth-sso.md:215]: I like your words a lot more! Thanks!

**Comment by wordsmither** [comment]
lgtm

**Pull request merged**